src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_kds.c

```diff
@@ -1920,9 +2014,9 @@ static int xocl_kds_update_xgq(struct xocl_dev *xdev, int slot_hdl,
 
 	num_cus = xocl_kds_fill_cu_info(xdev, slot_hdl, ip_layout, cu_info, MAX_CUS);
 
-	 /* The XGQ ERT doesn't support more than 64 CUs. Let this hardcoding.
-	  * We will re-looking at this once at supporting multiple xclbins.
-	  */
+	/* The XGQ ERT doesn't support more than 64 CUs. Let this hardcoding.
+	 * We will re-looking at this once at supporting multiple xclbins.
+	 */
 	if (num_cus > 64) {
```

Comment:
I know that this code isn't part of the PR, but it would be great to know what is so special about the magic number 64.

Reply:
The XGQ ERT doesn't support more than 64 CUs because of resource constraints. Added as a comment.